src/utils-lgpl/fcplay.c

```diff
@@ -1,7 +1,7 @@
-//SPDX-License-Identifier: LGPL-2.1-only
+/* SPDX-License-Identifier: LGPL-2.1-only */
 
-//Copyright (c) 2011-2012, Intel Corporation
```

Comment:
I am not really confused as to whether this is indeed an Intel contribution. I don't think we added anything with LGPL-only, did we?

Reply:
This was not but was derived from cplay.c (lots of code copy) hence the orignal copyright was retained by me and new one added

Reply:
I see. In that case, you should make it clearer. e.g.

Derived from cplay.c Copyright (c) 2011-2012, Intel Corporation